src/Microsoft.ML.Data/Transforms/ConvertTransform.cs

```diff
@@ -4,36 +4,58 @@
 using Microsoft.ML.Runtime.Data.Conversion;
+using Microsoft.ML.Runtime.EntryPoints;
 using Microsoft.ML.Runtime.Internal.Utilities;
 using Microsoft.ML.Runtime.Model;
 using Microsoft.ML.Runtime.Model.Onnx;
-using Microsoft.ML.Runtime.Command;
-using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.StaticPipe;
+using Microsoft.ML.StaticPipe.Runtime;
+using Microsoft.ML.Transforms;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
 
-[assembly: LoadableClass(ConvertTransform.Summary, typeof(ConvertTransform), typeof(ConvertTransform.Arguments), typeof(SignatureDataTransform),
+[assembly: LoadableClass(ConvertTransform.Summary, typeof(IDataTransform), typeof(ConvertTransform), typeof(ConvertTransform.Arguments), typeof(SignatureDataTransform),
     ConvertTransform.UserName, ConvertTransform.ShortName, "ConvertTransform", DocName = "transform/ConvertTransform.md")]
 
-[assembly: LoadableClass(ConvertTransform.Summary, typeof(ConvertTransform), null, typeof(SignatureLoadDataTransform),
+[assembly: LoadableClass(ConvertTransform.Summary, typeof(IDataTransform), typeof(ConvertTransform), null, typeof(SignatureLoadDataTransform),
     ConvertTransform.UserName, ConvertTransform.LoaderSignature, ConvertTransform.LoaderSignatureOld)]
 
-[assembly: LoadableClass(typeof(ConvertTransform.TypeInfoCommand), typeof(ConvertTransform.TypeInfoCommand.Arguments), typeof(SignatureCommand),
-    "", ConvertTransform.TypeInfoCommand.LoadName)]
+[assembly: LoadableClass(ConvertTransform.Summary, typeof(ConvertTransform), null, typeof(SignatureLoadModel),
+    ConvertTransform.UserName, ConvertTransform.LoaderSignature)]
+
+[assembly: LoadableClass(ConvertTransform.Summary, typeof(IRowMapper), typeof(ConvertTransform), null, typeof(SignatureLoadRowMapper),
+    ConvertTransform.UserName, ConvertTransform.LoaderSignature)]
 
 [assembly: EntryPointModule(typeof(TypeConversion))]
 
-namespace Microsoft.ML.Runtime.Data
+namespace Microsoft.ML.Transforms
 {
-    public sealed class ConvertTransform : OneToOneTransformBase
+    public static class TypeConversion
+    {
+        [TlcModule.EntryPoint(Name = "Transforms.ColumnTypeConverter", Desc = ConvertTransform.Summary, UserName = ConvertTransform.UserName, ShortName = ConvertTransform.ShortName)]
+        public static CommonOutputs.TransformOutput Convert(IHostEnvironment env, ConvertTransform.Arguments input)
+        {
+            Contracts.CheckValue(env, nameof(env));
+            env.CheckValue(input, nameof(input));
+
+            var h = EntryPointUtils.CheckArgsAndCreateHost(env, "Convert", input);
+            var view = ConvertTransform.Create(h, input, input.Data);
+            return new CommonOutputs.TransformOutput()
+            {
+                Model = new TransformModel(h, view, input.Data),
+                OutputData = view
+            };
+        }
+    }
+
+    public sealed class ConvertTransform : OneToOneTransformerBase
```

Comment:
>ConvertTransform [](start = 24, length = 16)

Let's get into a habit of renaming. `TypeConversionTransformer` / `TypeConversionEstimator`